operator/README.md

```diff
@@ -1,2 +1,104 @@
-# pmem-csi-operator
-Operator for PMEM-CSI
+<!-- TOC -->
+- [PMEM-CSI Operator](#pmem-csi-for-kubernetes)
+  - [About](#about)
+  - [Usage](#usage)
+    - [Pre-requisites](#pre-requisites)
+    - [Deploy PMEM-CSI operator](#deploy-pmem-csi-operator)
+    - [Deployment CRD API](#deployment-crd-api)
+  - [Example Deployment](#example-deployment)
+
+<!-- /TOC -->
+# PMEM-CSI-Operator
+
+## About
+
+Kubernetes operator for [PMEM-CSI](https://github.com/intel/pmem-csi) driver deployment. Using this operator one can easily deploy and manage PMEM-CSI driver in a Kubernetes cluster.
+
+The PMEM-CSI operator is based on the CoreOS [operator-sdk](https://github.com/operator-framework/operator-sdk) tools and APIs.
+
+## Usage
+
+### Pre-requisites
+
+A running Kubernetes cluster with one or more nodes installed with PMEM hardware(NVDIMM). The persistent memory on these nodes must be [pre-provisioned](../README.md#persistent-memory-pre-provisioning).
+
+### Deploy PMEM-CSI operator
+
+<!-- The assumptions:
+  a) pmem-csi-operator binary is part of released intel/pmem-csi-driver image and no additional steps required to build the images
+-->
+
+```sh
+$ kubectl create -f https://raw.githubusercontent.com/intel/pmem-CSI/operator/operator/deploy/operator.yaml
```

Comment:
`pmem-CSI/operator/operator/deploy/`
I think the path with double `operator` points to code in branch=operator, is that intentional to remain so in final version?

Reply:
I kept this as the PR is against the operator branch. But when merging to devel this should change.

Reply:
also, should pmem-CSI be without mixed uppercase, like pmem-csi

Reply:
> also, should pmem-CSI be without mixed uppercase, like pmem-csi

Fixed.